docs/Wdio_Utility.md

```diff
@@ -78,19 +78,46 @@ There are a few things to note about the webdriver.io configuration provided by
 - Test files should use `*-spec.js` naming format. The default spec search path is `./tests/wdio/**/*-spec.js`.
 - Use `/test_url_path` to direct test urls. This is appended to the `baseUrl` provided in the config.
 
-Then, to assist with testing, the TerraService provides the custom assertions `accessible` and `matchReference` and a Terra global helper to make testing easier:
 
-- `accessible()` validates the `axe()` accessibility assertions on the specified viewports are successful.
-- `matchReference()` validates the `checkElement` visual regression assertions on the specified viewports are successful.
+
+Then, to assist with testing, the TerraService provides custom methods to make testing easier:
```

Comment:
Then, to assist with testing, the TerraService provides the Terra global helper to make testing easier: